File: Modul-04/SOAL 1/PRAK401-2310817210009-MuhammadIbnuSina.py
print("Input: ")
input_str = input()
ANGKAKELIPATAN, SIMBOL = input_str.split()
ANGKAKELIPATAN = int(ANGKAKELIPATAN)

print("Output: ")
for i in range(1, 51):
    if i % ANGKAKELIPATAN == 0:
        print(SIMBOL, end=' ')
    else:
        print(i, end=' ')

File: Modul-04/SOAL 1/PRAK401-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int ANGKAKELIPATAN;
    char SIMBOL;

    printf("Input: \n");
    scanf("%d", &ANGKAKELIPATAN);
    scanf(" %c", &SIMBOL);

    printf("Output: \n");
    for (int i = 1; i <= 50; i++) {
        if (i % ANGKAKELIPATAN == 0) {
            printf("%c ", SIMBOL);
        } else {
            printf("%d ", i);
        }
    }

    return 0;
}
